src/groundStation/uTransceiver.py

```diff
@@ -0,0 +1,133 @@
+import ctypes
+from ctypes import cdll, c_int, c_char
+import sys
+import zmq
+import threading
+import os.path
+import socket
+import time
+
+
+
+# command format follows https://docs.google.com/spreadsheets/d/1zNhxhs0KJCp1187Vm3-zAzQHCY31f77l-0nlQmfXu1w/edit#gid=565953736
+# but with UHFDIR_ instead of UHF_ prefix. See document for examples
+
+#After sending a uhf-direct command, terminal displays all rx'd raw ASCII data for listentimeout_s
+
+class uTransceiver(object):
+
+    def __init__(self, opt):
+        self.listentimeout_s = 2.0
+        self.pipetimeout_s = 30.0
+        self.last_tx_time = 0
+        self.listen_en = False
+        self.pipe_en = False
+        self.u = opt
+        self.uhf = cdll.LoadLibrary("./ex2_uhf_software/uTransceiver.so")#consider making an env var for this?
+
+    def resetListenTimer(self):
+        self.listen_en = False
+    def listen(self):
+        port = 4321
+
+        context = zmq.Context()
+        socket = context.socket(zmq.SUB)
+        socket.connect("tcp://localhost:%s" % port)
+        socket.setsockopt(zmq.SUBSCRIBE, b"")
+        #socket.RCVTIMEO = 100
+
+        # Initialize poll set
+        poller = zmq.Poller()
+        poller.register(socket, zmq.POLLIN)
+        
+        # s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
```

Comment:
Commented code

Reply:
For testing purposes. Will add comment saying so.